shared/datastore.go

```diff
@@ -25,6 +25,23 @@ func LoadTestRun(ctx context.Context, id int64) (*TestRun, error) {
 	return &testRun, nil
 }
 
+// LoadTestRunsBySHAs loads all test runs that belong to any of the given revisions (SHAs).
+func LoadTestRunsBySHAs(ctx context.Context, shas ...string) (runs TestRuns, err error) {
+	for _, sha := range shas {
+		if len(sha) > 10 {
+			sha = sha[:10]
+		}
+		var shaRuns TestRuns
+		keys, err := datastore.NewQuery("TestRun").Filter("Revision =", sha).GetAll(ctx, &shaRuns)
+		if err != nil {
+			return runs, err
+		}
+		shaRuns.SetTestRunIDs(keys)
+		runs = append(runs, shaRuns...)
+	}
+	return runs, err
```

Comment:
Same sort of style question here, should we return err when we know it's nil?

Reply:
Nothing wrong with doing that (and it's preferred to return values (including nil) for all params)